snakemake/workflow.py

```diff
@@ -527,6 +527,8 @@ def execute(
         kubernetes=None,
         k8s_cpu_scalar=1.0,
         flux=None,
+        flux_operator=None,
+        flux_operator_ns=None,
```

Comment:
if flux_operator is combined with use_singularity, this should log a warning mentioning that it requires the cluster to run in priviledged mode, and that depending on the settings this might fail? Should also be mentioned in the docs on Snakemake's flux operator backend.